LICENSE.txt

```diff
@@ -24,4 +24,4 @@ DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
 SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
 CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
 OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
-OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```

Comment:
There is still an edit. This file shouldn't be part of the diff.

Reply:
I have checked again and the content of the license.txt is exactly that of license.txt in https://github.com/conda-forge/staged-recipes/blob/master/LICENSE.txt
Can you please have at a look at this once more.

Reply:
There is still this diff shown in the UI. It is only a single newline character but there shouldn't be an edit here.

Reply:
I have replaced the file with the one in master.
Now there should not be any difference